### PR TITLE
Fix/render props replace widget sockets

### PR DESCRIPTION
**Pull request by jeubank12**
react-hot-loader has issues with checking component types in the widgets package (https://github.com/gaearon/react-hot-loader#checking-element-types)

This could be fixed following react-hot-loader guidance but it can also be fixed with render props instead

### CODE REVIEW

**Comment by jeubank12** [comment]
`yarn storybook` fails with:
```
Cannot assign to read only property 'exports' of object '#<Object>'
TypeError: Cannot assign to read only property 'exports' of object '#<Object>'
    at Module.<anonymous> (http://localhost:3000/vendors~main.17e04e4c1b5a13de6c2d.bundle.js:257938:16)
    at Module../packages/ui-core/node_modules/react-select/node_modules/@babel/runtime/helpers/objectWithoutPropertiesLoose.js (http://localhost:3000/vendors~main.17e04e4c1b5a13de6c2d.bundle.js:257939:30)
    at __webpack_require__ (http://localhost:3000/runtime~main.17e04e4c1b5a13de6c2d.bundle.js:849:30)
    at fn (http://localhost:3000/runtime~main.17e04e4c1b5a13de6c2d.bundle.js:151:20)
    at Module.<anonymous> (http://localhost:3000/vendors~main.17e04e4c1b5a13de6c2d.bundle.js:257881:36)
    at Module../packages/ui-core/node_modules/react-select/node_modules/@babel/runtime/helpers/objectWithoutProperties.js (http://localhost:3000/vendors~main.17e04e4c1b5a13de6c2d.bundle.js:257903:30)
    at __webpack_require__ (http://localhost:3000/runtime~main.17e04e4c1b5a13de6c2d.bundle.js:849:30)
    at fn (http://localhost:3000/runtime~main.17e04e4c1b5a13de6c2d.bundle.js:151:20)
    at Module../packages/ui-core/node_modules/react-select/dist/react-select.browser.esm.js (http://localhost:3000/vendors~main.17e04e4c1b5a13de6c2d.bundle.js:256602:104)
    at __webpack_require__ (http://localhost:3000/runtime~main.17e04e4c1b5a13de6c2d.bundle.js:849:30)
```

File: packages/charts/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [2.5.21](https://github.com/keen/keen/compare/@keen.io/charts@2.5.20...@keen.io/charts@2.5.21) (2021-01-06)
```
Comment (maciejrybaniec) [inline packages/charts/CHANGELOG.md:6]: Are You maybe invoked a script responsible for bumping a versions ? This should be done automatically via Lerna in CI pipeline.

Comment (jeubank12) [reply packages/charts/CHANGELOG.md:6]: develop branch did not have the most recent changes in tag @keen.io/widgets@2.2.21 https://github.com/keen/keen/compare/develop...@keen.io/widgets@2.2.21

Comment (maciejrybaniec) [reply packages/charts/CHANGELOG.md:6]: Ok got it. The Storybook should works now - so we can right now test and check this PR.

Comment (jeubank12) [reply packages/charts/CHANGELOG.md]: @maciejrybaniec this is now rebased to the latest version of develop branch. I started storybook and confirmed that there are no visual regressions in the widgets

**Pull request merged**